src/Libraries/CoreNodeModels/DropDown.cs

```diff
@@ -235,23 +236,7 @@ public static string SaveSelectedIndexImpl(int index, IList<DynamoDropDownItem>
             }
 
             var item = items[index];
-            return string.Format("{0}:{1}", index, XmlEscape(item.Name));
-        }
-
-        protected static string XmlEscape(string unescaped)
```

Comment:
@QilongTang are these protected methods called from any of our derived classes? Might they be called from user's derived classes?

Reply:
They are indeed called from RevitDropDown.cs although only one instance, I originally plan to remove the same calls there using same refactor. But if we feel that removing such **protected** API could cause to 3rd party dev, I can instead simply update these two functions to use the new code I found work. e.g.
```
protected static string XmlEscape(string unescaped)
{
   return SecurityElement.Escape(unescaped);
}

protected static string XmlUnescape(string escaped)
{
  return System.Web.HttpUtility.HtmlDecode(escaped);
}
```

Reply:
@mjkkirschner @aparajit-pratap @sm6srw Let me know what you prefer, I am fine with either approach.

Reply:
@QilongTang I like the idea of keeping the methods and refactoring them as you show above. We can add a note that we must remove these in 3.0.

Reply:
+1